### ISSUE TITLE
{unicode: true} does not work for me

### ISSUE THREAD

**Issue by timsazon** [MikeKovarik/rage-edit#19]
Hi! I've tried the beta version of your library with Unicode option support but it doesn't work. Still, return ??? instead Unicode symbols.
Tested on W10 Version 10.0.17763 Build 17763
Example of code (value in registry `C:\Program Files\TeamSpeak 3 Client Реестр`):
```
async function getTeamSpeakDirectory() {
  let path = await Registry.get('HKLM\\Software\\TeamSpeak 3 Client', '', {unicode: true});
  if (!path) path = await Registry.get('HKCU\\Software\\TeamSpeak 3 Client', '');
  console.log(path); // C:\Program Files\TeamSpeak 3 Client ??????
  return path;
}
```
It looks like 'cmd.exe /c chcp.com 67001' does not change encoding for next call,
 `await Registry.getCodePage()` return 437 after `await Registry.setCodePage(67001)`.

**Comment by OddMorning** [comment]
That's really strange. I just checked your code on Win7, Win10 1803, and Win10 1903 (the May 2019 one) and it worked as intended.

Can you try to use `await Registry.enableUnicode()` before `await Registry.get(<…>)` instead of `{unicode: true}`? Or try to manually enter `.setCodePage(65001)` / `cmd.exe /c chcp.com 65001` instead of `67001`.

**Comment by timsazon** [comment]
Yes, I've already tried (67001 was just a typo in the issue), nothing changes. Btw, I stopped at this solution with `regedit` library and it's fine:
```
export async function getRegistryValue(path, key) {
  return await new Promise(resolve => {
    regedit.list(path, function (err, result) {
      if (err) return resolve(undefined);
      resolve(result[path].values[key] ? result[path].values[key].value : undefined);
    });
  })
}
```
